lib/rubocop/cops_documentation_generator.rb

```diff
@@ -114,7 +114,15 @@ def safety_object(safety_objects, cop)
   def required_ruby_version(cop)
     return '' unless cop.respond_to?(:required_minimum_ruby_version)
 
-    "NOTE: Required Ruby version: #{cop.required_minimum_ruby_version}\n\n"
+    if cop.required_minimum_ruby_version.nonzero?
+      note = "Required Ruby version: #{cop.required_minimum_ruby_version}"
+    elsif cop.required_maximum_ruby_version.finite?
+      note = "Maximum Ruby version: #{cop.required_maximum_ruby_version}"
```

Comment:
This reads better to me. WDYT?

```suggestion
      note = "Required Ruby version: <= #{cop.required_maximum_ruby_version}"
```

Reply:
Should minimum have a `>=` then?

Reply:
For correctness, I guess. But I think it would just be visual clutter.

If you have both an upper and lower bound I'd do it like `>=2.7, <=3.3` but in the most common case I would not add it.

Reply:
Updated. I'm leaving out the min+max case for now but we might have to handle that in the future.

Reply:
There are some cops where this will be useful, so this is a welcome addition. I'll dig them up after this is merged